modules/sponsor/SponsorForm.tsx

```diff
@@ -157,6 +183,7 @@ const SponsorForm = (): ReactElement => {
                               className={classes.textField}
                               variant="outlined"
                               label="First Name"
+                              inputProps={{"data-testid": "sponsor-form-first-name"}}
```

Comment:
I like using `data-testid` for simplicity when doing up e2e tests. There's a debate that using accessibility form elements are better but at work we've found that a bit hard to maintain so I like having these elements to hook into, and then testing their accessible names which I'm not really doing ATM but is something I want to add to the e2e tests.